src/main/java/com/ithillel/service/appcontext/InjectorApplicationContext.java

```diff
@@ -0,0 +1,55 @@
+package com.ithillel.service.appcontext;
+
+import com.ithillel.service.annotation.CustomBean;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class InjectorApplicationContext implements ApplicationContext {
+
+    private static Map<String, Object> beans = new HashMap<>();
+
+    public InjectorApplicationContext() throws ClassNotFoundException {
+        List<Class<?>> classes = addClasses("src\\main\\java\\com\\ithillel");
```

Comment:
Be careful with "\\" it can work inn Windows, but not in Linux/Max OS, instead of \\ you can use File.separator 
smng likes that 
String.format("src%smain%sjava%scom%sithillel", File.separator, File.separator, File.separator, File.separator))

Reply:
Anyway, if yo want to make it much pretty you can check

` Reflections reflections = new Reflections("com.ithillel", new SubTypesScanner(false));
 return  reflections.getSubTypesOf(Object.class);`

just add in pom
        <dependency>
            <groupId>org.reflections</groupId>
            <artifactId>reflections</artifactId>
            <version>0.9.11</version>
        </dependency>

Up to you

as additional please rename method from addClasses to getClasses;